.php_cs.dist

```diff
@@ -4,7 +4,7 @@ return PhpCsFixer\Config::create()
     ->setRules(array(
```

Comment:
So this file, `.php_cs.dist` is new to me. Can you explain how you would make use of it?